src/pages/Admin/Admin_v2.tsx

```diff
@@ -50,7 +47,7 @@ const Admin_v2: React.FC = () => {
     <>
       <S.Wrapper className="no-venue-selected">
         <S.ViewWrapper>
-          <AdminVenues venues={venues} />
+          <AdminVenues venues={relatedVenues} />
```

Comment:
This should be a list of partymap venues that belong to you (you are owner). I don't think there is a hook or something else that returns such list, that's why we had all of the venues passed in and then the component `AdminVenues` filters them.

If you are to adapt `relatedVenues` to pass these owner venues you should also add `withRelatedVenues` to the subrouter.

Reply:
Replaced with `useAdministeredVenues()` that returns `administeredVenues` instead of `relatedVenues`  in f97606d53e7ceba1ab4ad0433711a96c16d5b7ea

Reply:
Can we Calle it useOwnedVenues or similar please

Reply:
Makes no difference to me if `useOwnedVenues` or `useAdministeredVenues`, I was following the `useAdminVenues` prior art which  https://github.com/sparkletown/sparkle/pull/1620/commits/103baa1a25f079ee2d0ccf9158c295a66ee594c9 removes as redundant.

Just in case, will RFC @denisdimitrov before doing the rename.

Reply:
renamed in ef8651a789a447dbfc954eb4e516ff7d298adeea